manageiq-loggers.gemspec

```diff
@@ -1,25 +1,22 @@
-
-lib = File.expand_path("../lib", __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "manageiq/loggers/version"
+require "rbconfig"
+require_relative "lib/manageiq/loggers/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "manageiq-loggers"
-  spec.version       = Manageiq::Loggers::VERSION
-  spec.authors       = ["ManageIQ Authors"]
-
-  spec.summary       = %q{Loggers for ManageIQ projects}
-  spec.homepage      = "https://github.com/ManageIQ/manageiq-loggers"
-  spec.licenses      = ["Apache-2.0"]
+  spec.name     = "manageiq-loggers"
+  spec.version  = Manageiq::Loggers::VERSION
```

Comment:
Interesting?  This should have been `ManageIQ`  (capital IQ).  Is that bug in the gem?  cc @agrare @bdunne

Reply:
Good catch. Looks like a bug to me.

Reply:
https://github.com/ManageIQ/manageiq-loggers/pull/13